docs/txp.md

```diff
@@ -296,12 +301,25 @@ Stakes modification is a bit more complex, but is also very
 intuitive. We have `TxOut`s corresponding to inputs and outputs of
 transaction. Each `TxOut` can be converted to a list of
 `(StakeholderId, Coin)` pairs. It depends on `AddrStakeDistribution`
-of given address. For `BootstrapEraDistr` the value of `TxOut` is
-distributed among bootstrap stakeholders proportional to their weights.
-If distribution is `SingleKeyDistr id`, then the value of `TxOut` will
-be assigned to `id`. In case of `MultiKeyDistr` stake will be
-distributed among multiple stakeholders according to specified
-portions. These lists are concatenated and we have two lists of
+of given address.
+* For `BootstrapEraDistr` the value of `TxOut` is distributed among
+bootstrap stakeholders proportional to their weights. The behavior
+depends on whether the value is less than the sum of weights of all
+stakeholders (this sum is called `bootDustThreshold`). If it's greater
+than or equal to `bootDustThreshold`, then each stakeholder receives
+`val / weights_sum` coins (`val` is the value of `TxOut`) and one
```

Comment:
I don't get why you mentioned:
>`TxOut` is distributed among bootstrap stakeholders proportional to their weights.

but 
>`val / weights_sum` coins

it should be `val * w_i / weights_sum` coins

Reply:
Right.